packages/validators/index.d.ts

```diff
@@ -32,23 +32,23 @@ export const minLength: (
 ) => ValidationRuleWithParams<{ length: number }>;
 export const minValue: (
   min: number | Ref<number> | string | Ref<string>
-) => ValidationRuleWithParams<{ min: number }>
-export const not: (validator: ValidationRule) => ValidationRuleWithoutParams
-export const numeric: ValidationRuleWithoutParams
-export const or: (
-  ...validators: ValidationRule[]
-) => ValidationRuleWithoutParams
-export const required: ValidationRuleWithoutParams
-export const requiredIf: (prop: boolean | string | (() => boolean | Promise<boolean>)) => ValidationRuleWithoutParams
-export const requiredUnless: (prop: boolean | string | (() => boolean | Promise<boolean>)) => ValidationRuleWithoutParams
+) => ValidationRuleWithParams<{ min: number }>;
+export const not: <T = unknown>(validator: ValidationRule<T>) => ValidationRuleWithoutParams;
+export const numeric: ValidationRuleWithoutParams;
+export const or: <T = unknown>(
+  ...validators: ValidationRule<T>[]
+) => ValidationRuleWithoutParams;
+export const required: ValidationRuleWithoutParams;
+export const requiredIf: (prop: boolean | string | (() => boolean | Promise<boolean>)) => ValidationRuleWithoutParams;
+export const requiredUnless: (prop: boolean | string | (() => boolean | Promise<boolean>)) => ValidationRuleWithoutParams;
 export const sameAs: <E = unknown>(
   equalTo: E,
   otherName?: string
 ) => ValidationRuleWithParams<{ equalTo: E, otherName: string }>;
 export const url: ValidationRuleWithoutParams;
 export const helpers: {
-  withParams: <T = unknown>(params: object, validator: ValidationRule<T>) => ValidationRuleWithParams
-  withMessage: <T = unknown>(message: string | ((params: MessageProps) => string), validator: ValidationRule<T>) => ValidationRuleWithParams
+  withParams: (params: object, validator: ValidationRule) => ValidationRuleWithParams
+  withMessage: (message: string | Function, validator: ValidationRule) => ValidationRuleWithParams
```

Comment:
Why are we removing the more detailed declaration for this? Is it better to just have a function?

Reply:
This change was part of the original PR, I think it was done before you updated the types in the main branch. I'm not sure what the intention was. I can't think of a time that I've ever used the `Function` keyword like that and don't want to speculate so I'm just going to update these to match main.